go/cilium/api/bpf_metadata.pb.go

```diff
@@ -36,13 +36,16 @@ type BpfMetadata struct {
 	IsIngress bool `protobuf:"varint,2,opt,name=is_ingress,json=isIngress,proto3" json:"is_ingress,omitempty"`
 	// Use of the original source address requires kernel datapath support which
 	// may or may not be available. 'true' if original source address
-	// functionality is availeble. Original source address use may still be
+	// should be used. Original source address use may still be
 	// skipped in scenarios where it is knows to not work.
-	MayUseOriginalSourceAddress bool `protobuf:"varint,3,opt,name=may_use_original_source_address,json=mayUseOriginalSourceAddress,proto3" json:"may_use_original_source_address,omitempty"`
-	// Set source endpoint ID in socket mark instead of source security ID on
-	// upstream connections. This is needed for L7 LB listeners. Only valid for
-	// egress.
-	EgressMarkSourceEndpointId bool `protobuf:"varint,4,opt,name=egress_mark_source_endpoint_id,json=egressMarkSourceEndpointId,proto3" json:"egress_mark_source_endpoint_id,omitempty"`
+	UseOriginalSourceAddress bool `protobuf:"varint,3,opt,name=use_original_source_address,json=useOriginalSourceAddress,proto3" json:"use_original_source_address,omitempty"`
+	// True if the listener is used for an L7 LB. In this case policy enforcement is done on the
+	// destination selected by the listener rather than on the original destination address. For
+	// local sources the source endpoint ID is set in socket mark instead of source security ID if
+	// 'use_original_source_address' is also true, so that the local source's egress policy is
+	// enforced on the bpf datapath.
+	// Only valid for egress.
+	IsL7Lb bool `protobuf:"varint,4,opt,name=is_l7lb,json=isL7lb,proto3" json:"is_l7lb,omitempty"`
```

Comment:
with the rename here, can we still backport this change to the older releases (e.g. cilium 1.13 with envoy 1.23). Normally, we didn't update API in old releases.

Reply:
Old versions can use the old API definitions with old names.

Reply:
So the backport would be different in that regard.

Reply:
Or then we can also backport the name change, it's not really a big deal since we do not go older than Cilium 1.13.

Reply:
yeah, i was thinking about backport this to 1.12 as well. But we can adapt the patch as well.